dask/array/tests/test_optimization.py

```diff
@@ -263,24 +263,6 @@ def test_dont_fuse_numpy_arrays():
         assert sum(isinstance(v, np.ndarray) for v in dsk.values()) == 1
 
 
-def test_minimize_data_transfer():
```

Comment:
I think this test is meaningless now. There is no "big_key" anymore.

Reply:
I think that this test was likely intended to represent other cases, for example where `x` is a zarr array.  We don't want to include the zarr array in every task directly (this causes many non-trivially large tasks), but would rather have one big task.

Previously we used numpy as a stand-in array for this case, but now that we're special-casing numpy that no longer works.  We could choose a another stand-in, like `np.matrix`, or use zarr directly, with a `zarr = pytest.importorskip('zarr')` import.

Reply:
Given the churn around handling types, my preference is currenty to use Zarr, if that's convenient.

Also, my apologies for not catching this part of the PR earlier.  It's always frustrating when they're always just-one-more-thing in a PR :/  I think I fail the "Don’t let reviewers drag things out too long" test here: http://matthewrocklin.com/blog/2019/05/18/maintainer

Reply:
:+1: I've switched to a zarr array. np.matrix raises a warning which makes the test fail...

Reply:
> Also, my apologies for not catching this part of the PR earlier.

no worries!